app/assets/javascripts/controllers/container_dashboard/container_dashboard_controller.js

```diff
@@ -61,7 +61,8 @@ miqHttpInject(angular.module('containerDashboard', ['ui.bootstrap', 'patternfly'
           id = '';
         }
         else {
-          id = '/'+ (/\/(\d+)/.exec($location.absUrl())[1]);
+          // search for http://<url>/<controller>/<number>...
+          id = '/' + (/\/.+\/.+\/(\d+)/.exec($location.absUrl())[1]);
```

Comment:
@himdel 
a. i wanted to use `$location.path()` but it did not work for me here :-( , is there something i'm missing ?
b. can i use `location.pathname` here ?